examples/sample_measurement/SampleMeasurement.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.SampleMeasurement_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
```

Comment:
We should update the template files to include these standard annotations and suggest how the user should fill them in. We should also use the existing variables like 'display name' to fill them in when generated or add them as optional command-line parameters.

Reply:
This can and probably should be done in a separate PR.